deploy/src/deployment-servicemesh.yaml

```diff
@@ -13,56 +13,57 @@ spec:
-
-        olm.relatedImage.v2_1.cni: registry.redhat.io/openshift-service-mesh/istio-cni-rhel8:2.1.3
-        olm.relatedImage.v2_1.grafana: registry.redhat.io/openshift-service-mesh/grafana-rhel8:2.1.3
-        olm.relatedImage.v2_1.pilot: registry.redhat.io/openshift-service-mesh/pilot-rhel8:2.1.3
-        olm.relatedImage.v2_1.prometheus: registry.redhat.io/openshift-service-mesh/prometheus-rhel8:2.1.3
-        olm.relatedImage.v2_1.proxyv2: registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:2.1.3
-        olm.relatedImage.v2_1.wasm-cacher: registry.redhat.io/openshift-service-mesh/pilot-rhel8:2.1.3
-        olm.relatedImage.v2_1.rls: registry.redhat.io/openshift-service-mesh/ratelimit-rhel8:2.1.3
-
-        olm.relatedImage.v2_2.cni: registry.redhat.io/openshift-service-mesh/istio-cni-rhel8:2.2.0
-        olm.relatedImage.v2_2.grafana: registry.redhat.io/openshift-service-mesh/grafana-rhel8:2.2.0
-        olm.relatedImage.v2_2.pilot: registry.redhat.io/openshift-service-mesh/pilot-rhel8:2.2.0
-        olm.relatedImage.v2_2.prometheus: registry.redhat.io/openshift-service-mesh/prometheus-rhel8:2.2.0
-        olm.relatedImage.v2_2.proxyv2: registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:2.2.0
-        olm.relatedImage.v2_2.wasm-cacher: registry.redhat.io/openshift-service-mesh/pilot-rhel8:2.2.0
-        olm.relatedImage.v2_2.rls: registry.redhat.io/openshift-service-mesh/ratelimit-rhel8:2.2.0
+        # 1.0 images
+        olm.relatedImage.v1_0.3scale-istio-adapter: registry-proxy.engineering.redhat.com/rh-osbs/openshift-service-mesh-3scale-istio-adapter-rhel8:1.0.0
+        olm.relatedImage.v1_0.citadel: registry-proxy.engineering.redhat.com/rh-osbs/openshift-service-mesh-citadel-rhel8:${OSSM_10}
+        olm.relatedImage.v1_0.cni: registry-proxy.engineering.redhat.com/rh-osbs/openshift-service-mesh-istio-cni-rhel8:${OSSM_10}
+        olm.relatedImage.v1_0.galley: registry-proxy.engineering.redhat.com/rh-osbs/openshift-service-mesh-galley-rhel8:${OSSM_10}
+        olm.relatedImage.v1_0.grafana: registry-proxy.engineering.redhat.com/rh-osbs/openshift-service-mesh-grafana-rhel8:${OSSM_10}
+        olm.relatedImage.v1_0.mixer: registry-proxy.engineering.redhat.com/rh-osbs/openshift-service-mesh-mixer-rhel8:${OSSM_10}
+        olm.relatedImage.v1_0.pilot: registry-proxy.engineering.redhat.com/rh-osbs/openshift-service-mesh-pilot-rhel8:${OSSM_10}
+        olm.relatedImage.v1_0.prometheus: registry-proxy.engineering.redhat.com/rh-osbs/openshift-service-mesh-prometheus-rhel8:${OSSM_10}
+        olm.relatedImage.v1_0.proxy-init: registry-proxy.engineering.redhat.com/rh-osbs/openshift-service-mesh-proxy-init-rhel7:${OSSM_10}
+        olm.relatedImage.v1_0.proxyv2: registry-proxy.engineering.redhat.com/rh-osbs/openshift-service-mesh-proxyv2-rhel8:${OSSM_10}
+        olm.relatedImage.v1_0.sidecar-injector: registry-proxy.engineering.redhat.com/rh-osbs/openshift-service-mesh-sidecar-injector-rhel8:${OSSM_10}
```

Comment:
Shouldn't we remove images for v1.0, since we [removed support for installing v1.0](https://github.com/maistra/istio-operator/pull/915)?

Reply:
I think you're right- in this PR I'm just synchronizing the CSVs though. I can create a follow-up that removes them